File: server/src/com/inkostilation/pong/server/application/Time.java
package com.inkostilation.pong.server.application;

public class Time {

    private static long lastTime = System.nanoTime();

    public static float getDeltaTime() {
        long time = System.nanoTime();
        long delta = time - lastTime;
        return delta / 1000000000f;
    }

    public static void updateTime() {
        lastTime = System.nanoTime();
    }
}

File: server/src/com/inkostilation/pong/server/application/Time.kt
package com.inkostilation.pong.server.application

object Time {
    private var lastTime = System.nanoTime()
    val deltaTime: Float
        get() {
            val delta = System.nanoTime() - lastTime
            return delta / 1000000000f
        }

    fun updateTime() {
        lastTime = System.nanoTime()
    }
}